package.json

```diff
@@ -164,7 +163,7 @@
 		"whatwg-fetch": "1.1.1"
 	},
 	"devDependencies": {
-		"@automattic/calypso-build": "1.0.0-alpha.0",
+		"@automattic/calypso-build": "1.0.0-alpha.2",
 		"@automattic/calypso-color-schemes": "1.0.0-alpha.1",
```

Comment:
is this `@automattic/calypso-color-schemes` anymore used in Jetpack directly as it's now also [dependency in calypso-build](https://github.com/Automattic/jetpack/pull/11802/files#diff-8ee2343978836a779dc9f8d6b794c3b2R10)?

(asked the same [just elsewhere](https://github.com/Automattic/jetpack/pull/11941/files#r272532005) accidentally)

Reply:
Good spot! I'll remove.

Reply:
Just noting that actually we might want to bring this dependency back at some point, since looks like it would be useful for Calypsoify module:

https://github.com/Automattic/jetpack/blob/4884963a1049378290f0b35584084f251075b168/modules/calypsoify/_colors.scss